test/__bootstrap__/expect.assertions.js

```diff
@@ -0,0 +1,40 @@
+//
+// Partially adapted from jasmine-expect-count
+// https://github.com/UziTech/jasmine-expect-count/blob/master/LICENSE
+//
+EXPECT_PATCHING_SCOPE: {
+    if (!globalThis.jasmine) {
+        throw new Error("jasmine must be loaded before patching in expect.assertions(x)");
+    }
+
+    let actualExpects = 0;
+    let expectedExpects = null;
+
+    beforeEach(() => {
+        actualExpects = 0;
+        expectedExpects = null;
+    });
+
+    let originalExpect = globalThis.expect;
+
+    globalThis.expect = function(...args) {
+        actualExpects++;
+        return originalExpect.apply(this, args);
```

Comment:
`Reflect.apply(originalExpect, this, args)` :)

Reply:
I’ll roll in all of these suggestions. The current code is mostly just from the original module, and I definitely don’t mind using the more modern capabilities. 😁